src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+
+    private List<Answer> answers = new ArrayList<>();
+
+    public void add(Answer answer) {
+        this.answers.add(answer);
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+
+    public void validateAnswerOwner(NsUser loginUser) throws CannotDeleteException {
+        for (Answer answer : this.answers) {
+            answer.validateAnswerOwner(loginUser);
+            answer.updateDeleted();
+        }
+    }
+
+    public void addDeleteAnswerHistory(List<DeleteHistory> deleteHistories) {
```

Comment:
```suggestion
    public List<DeleteHistory> toDeleteAnswerHistories() {
        List<DeleteHistory> deleteHistories = new ArrayList<>();
```
개인적으로 List 콜렉션을 인자로 전달해 저장하는 것보다 위와 같이 해당 메서드에서 직접 생성해 반환하는 것이 다른 객체와의 의존 관계가 발생하지 않기 때문에 선호하는 방법임